scripts/lua/if_stats.lua

```diff
@@ -1321,7 +1325,7 @@ $("#network").change(function() {
 
 local shapers = shaper_utils.getSortedShapers(ifid)
 
-function print_network_shapers(shapers, curshaper_id, terminator)
+function print_shapers(shapers, curshaper_id, terminator)
```

Comment:
It's really visualization bound, I wouldn't put it into a util file. In future, all the policing related code should be moved entirely to the pro side